src/clarity-angular/layout/tabs/tab-link.ts

```diff
@@ -5,34 +5,48 @@
  */
 import {
     Component,
-    Input, forwardRef, Inject
+    forwardRef,
+    HostListener,
+    Inject,
+    Input
 } from "@angular/core";
+import {IfActiveService} from "../../utils/conditional/if-active.service";
+import {IF_ACTIVE_ID} from "./tab";
 
-import { Tabs } from "./tabs";
+let nbTabLinkComponents: number = 0;
 
 @Component({
     selector: "clr-tab-link",
-    templateUrl: "./tab-link.html",
+    template: `
+        <button class="btn btn-link nav-link" type="button">
+            <ng-content></ng-content>
+        </button>
```

Comment:
This is definitely doable. I just tested it real quick and the only problem is our css doesn't work as is since we assume a level of hierarchy that disappears with this approach. I can't guarantee how clean the changed css will be, especially since I assume we're gonna try to preserve backward compatibility for all navs using the same css. Do you still want me to try?